### ISSUE TITLE
Gloomspite Gitz Missing Battalions From White Dwarf

### ISSUE THREAD

**Issue by PretendObject** [BSData/warhammer-age-of-sigmar#1788]
White Dwarf #455: The Jaws of Mork is missing (https://www.goonhammer.com/white-dwarf-455-the-jaws-of-mork/)

White Dwarf #457 The Tome Celestial: Troggherds is missing (https://www.goonhammer.com/white-dwarf-457-the-tome-celestial-troggherds/)

White Dwarf # 459 The Tome Celestial: The Grimscuttle Tribes is missing (https://www.goonhammer.com/white-dwarf-459-the-tome-celestial-the-grimscuttle-tribes/)

**Comment by abraracurcix** [comment]
If you are using the App you must click on the "Gloomspite Gitz" Allegiance to select the subfaction.
